acpi/src/fadt.rs

```diff
@@ -127,4 +127,31 @@ impl Fadt {
             other => PowerProfile::Reserved(other),
         }
     }
+
+    pub fn pm_timer_block(&self) -> Result<Option<GenericAddress>, AcpiError> {
+        let raw = unsafe {
+            self.x_pm_timer_block.access(self.header().revision).or_else(|| {
+                if self.pm_timer_block != 0 {
+                    Some(RawGenericAddress {
+                        address_space: 0,
+                        bit_width: 0,
+                        bit_offset: 0,
+                        access_size: self.pm_timer_length,
+                        address: self.pm_timer_block.into(),
+                    })
+                } else {
+                    None
+                }
+            })
+        };
+
+        match raw {
+            Some(raw) => Ok(Some(GenericAddress::from_raw(raw)?)),
+            None => Ok(None),
+        }
+    }
+
+    pub fn flags(&self) -> u32 {
```

Comment:
Since this will be public, could we call it `raw_flags`, as we may want to provide a nicer view into the flags in the future, and it would be nice for that to be called `flags`?

Reply:
(as mentioned below) as this is just a field access, I wouldn't be against just making the field `pub`, and then we can add a `flags` method later on for parsing the bitfield if we want?